extra-network/cgproxy/spec

```diff
@@ -0,0 +1,4 @@
+VER=0.19
+SRCTBL="https://github.com/springzfx/cgproxy/archive/v$VER.tar.gz"
```

Comment:
Please use the new [spec file format](https://wiki.aosc.io/developer/packaging/acbs/spec-format/). You have used it for the other package, so why not here?